docs/flink_processor.md

```diff
@@ -6,12 +6,27 @@ sections we describe how to use the Flink processor.
 ## Deployment Mode
 The Flink processor runs the Flink job in one of the following mode:
 
+- None
 - Session Mode
 - Kubernetes Application Mode
 
 Session mode is the default mode. User can specify the deployment mode with 
 configuration `deployment_mode`.
 
+### None
+
+If the deployment mode is set to "none", it means that Feathub will not submit the flink
+job to a remote Flink cluster, instead it executes the Flink job with the Flink 
+execution environment from the context where the program is executed. It is up to the
+user to submit the Feathub job as the Flink job entrypoint to the remote Flink cluster. 
+For example, you can use the Flink cli to submit the Feathub job to a remote Flink
+cluster. Or you can submit the Feathub job to the [Alibaba Cloud Realtime Compute for 
```

Comment:
It might be simpler to describe general FeatHub configurations w.r.t. Apache Flink rather than vendor-specific products. We can add vendor-specific content in separate folders.